src/service/FlywheelService/TransactionController.ts

```diff
@@ -130,4 +130,27 @@ export class TransactionController extends APIClient {
             }
         );
     }
+
+    /**
+     * TODO: Please comment this method
+     * @param transactionId
+     * @param targetPhase
+     */
+    async moveToPhase(transactionId: number, targetPhase: number) {
+        return this.invokeApiWithErrorHandling(
```

Comment:
Does it not return anything ? 

```suggestion
        return this.invokeApiWithErrorHandling<???>(
```

Reply:
void